### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by pmig**
Closes # 

## 📑 Description
Text input mostly comes from our "new" website and @thekubernaut 

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

**Review by thekubernaut — APPROVED** [review]

**Comment by pmig** [comment]
The last one to approve should press the merge button =)

File: README.md [hunk 1]
```diff
@@ -34,60 +34,53 @@
...
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| **Focusing on simplicity and reliability with our CLI and UI** <br> Easily install packages in your cluster via the Glasskube UI, where all packages are conveniently located, eliminating the need to search for a Helm repository.                              | ![Glasskube GUI](https://github.com/glasskube/glasskube/assets/3041752/323994d6-6b08-4dca-ac59-d29ae6b37f94) |
+| **Package configurations** <br> Configure packages with typesafe input values via the UI or interactive CLI questionnaire. Inject values from other packages, ConfigMaps, and Secrets easily. No more untyped and undocumented values.yamlfiles.                  | ![Configuration](https://github.com/glasskube/glasskube/assets/3041752/df6bd7d4-7cac-435b-b3a0-31c3cab6069b) |
```
Comment (jakepage91) [inline README.md:77]: Missing a space here: _values.yamlfiles_ => values.yaml files

File: README.md [hunk 2]
```diff
@@ -34,60 +34,53 @@
...
+| **Dependency Management** <br> Glasskube packages are dependency aware, so they can used and referenced by multiple other packages. They will also get installed in the correct namespace. This is how umbrella charts should have worked from the beginning.     | ![Dependency](https://github.com/glasskube/glasskube/assets/3041752/9588b3fc-2a87-454e-97ff-b0f7558717bc)    |
```
Comment (jakepage91) [inline README.md:78]: so they can _be_ used and referenced by multiple other packages

File: README.md [hunk 3]
```diff
@@ -34,60 +34,53 @@
...
+| **Reactions and comments** <br> Discuss and upvote your favorit Kubernetes package on [GitHub](https://github.com/glasskube/glasskube/discussions/categories/packages) or right inside the Glasskube UI.                                                          | ![Reactions](https://github.com/glasskube/glasskube/assets/3041752/56f08373-fbbe-46fd-820e-fb637114336b)     |
+| **GitOps Integration** <br> All Glasskube packages are custom resources, manageable via GitOps. We're also integrating with [renovate](https://github.com/renovatebot/renovate/issues/29322)                                                                      | ![GitOps](https://github.com/glasskube/glasskube/assets/3041752/8c359e61-9eec-4413-9c13-bca5cd8710d1)        |
+| **Multiple Repositories and private packages** <br> Use multiple repositories and publish your own private packages. This could be your companies Internal services packages, so all developers will have the up-to-date and easily configured internal services. | ![Repo](https://github.com/glasskube/glasskube/assets/130456438/e2f4472b-5b80-4043-9c78-9ccabd8f3337)        |
```
Comment (jakepage91) [inline README.md:82]: Any reason why _Internal_ starts with a capitol `I`?

Also 

Remove the `the` from _the up-to-date_

**Review by jakepage91 — COMMENTED** [review]
Just have a couple of remarks here.

**Review by jakepage91 — APPROVED** [review]
LGTM!

**Review by christophenne — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -34,60 +34,53 @@
...
-- 🛠️ **GitOps Ready** with ArgoCD or Flux:
-  <br> Seamlessly integrate Glasskube into your GitOps workflow with support for popular tools like ArgoCD or Flux.
-- 📦 **Central Package Repository**:
-  <br> Keep track of all your packages in one central repository, with a [planned](https://glasskube.dev/roadmap/?utm_source=github) feature for custom repositories.
-- 🔍 **Cluster Scan** ([planned](https://glasskube.dev/roadmap/?utm_source=github)):
-  <br> Introducing the Cluster Scan feature in a future version, which allows you to detect packages in your cluster, providing valuable insights for better management and upgrade paths.
-- 🔐 **Version Pinning**:
-  <br> With a future version, Glasskube will introduce Version Pinning, giving you control over your package versions.
+In contrast, tools like Homebrew, apt, and dnf felt easy to use and rarely caused problems.
+While working on other cloud-native projects, users consistently highlighted several common pain points.
```
Comment (kosmoz) [inline README.md:56]: I think this sentence could be expressed more clearly. Does it mean:
1. "_We_ worked on other projects and _our_ users had problems", or
2. "Users working on _their_ projects had problems"?

I think the first interpretation makes more sense from the context but the sentence reads more like the second, so I was confused.

File: README.md [hunk 5]
```diff
@@ -34,60 +34,53 @@
...
 - [Need help?](https://github.com/glasskube/glasskube#-need-help)
 - [Related projects](https://github.com/glasskube/glasskube#-related-projects)
 - [How to Contribute](https://github.com/glasskube/glasskube#-how-to-contribute)
-- [Supported by](https://github.com/glasskube/glasskube#-supported-by)
+- [Supported by](https://github.com/glasskube/glasskube#-thanks-to-all-our-contributors)
+- [Activity](https://github.com/glasskube/glasskube#-activity)
+- [License](https://github.com/glasskube/glasskube#-license)
+
+## ✨ Features
+|                                                                                                                                                                                                                                                                  |                                                                                                              |
```
Comment (kosmoz) [inline README.md:74]: empty table row?
image

File: README.md [hunk 6]
```diff
@@ -110,9 +103,13 @@ After successfully bootstrapping your cluster, you are ready to start the packag
 glasskube serve
 ```
 
-This command will open [`http://localhost:8580`](http://localhost:8580) in your default browser. 
+This command will open [`http://localhost:8580`](http://localhost:8580) in your default browser.
 Congratulations, you can now explore and install all our available packages! 🎉
 
+## 🎬 How to install you first package
+
+![cast](https://github.com/glasskube/glasskube/assets/16959694/f8b936ca-7b58-4e2b-8845-17da089f2384)
```
Comment (kosmoz) [inline README.md:111]: This is starting to feel out of place, in addition to being a seriously outdated demo. IMO we should remove it and make coming up with an alternative a priority.

File: README.md [hunk 7]
```diff
@@ -34,60 +34,53 @@
...
-![Glasskube GUI Mockup](https://github.com/glasskube/operator/assets/3041752/71d0da0c-34ac-40b7-8740-bd2a81ca9f07)
+![Glasskube GUI](https://github.com/glasskube/glasskube/assets/3041752/323994d6-6b08-4dca-ac59-d29ae6b37f94)
 
 
 ## 📦 What is Glasskube?
 
-Glasskube is the **next generation Kubernetes package manager**, now available in its `beta version`. Glasskube empowers you to effortlessly **install, upgrade, configure, and manage** your Kubernetes cluster packages, all while streamlining repetitive and cumbersome maintenance tasks. We're excited to roll out the beta release and invite all users to test it out! **Your feedback is incredibly valuable to us** as we aim to shape Glasskube into `the best Kubernetes package manager out there`. So go ahead, dive in, play around, and shoot us your thoughts!
-
+Glasskube is an **Open Source package manager for Kubernetes**.
+It makes deploying, updating, and configuring packages on Kubernetes **20 times faster** than tools like **Helm or Kustomize**.
```
Comment (kosmoz) [inline README.md:46]: Wild claim. A side-by-side demo would be cool to prove this.

**Review by kosmoz — CHANGES_REQUESTED** [review]
Some suggestions and general comments

File: README.md (hunk 6 above)
Comment (pmig) [reply README.md:111]: @jakepage91 is already working on the alternative - as soon as it is ready I will exchange it in a follow up pr.

File: README.md (hunk 5 above)
Comment (kosmoz) [reply README.md:74]: unlucky

**Review by kosmoz — APPROVED** [review]

**Review by kosmoz — APPROVED** [review]

**Pull request merged**